src/nri/sampler.go

```diff
@@ -72,6 +70,15 @@ func (cs *ContainerSampler) SampleAll(ctx context.Context, i *integration.Integr
 	}
 
 	for _, container := range containers {
+		metrics, err := cs.metrics.Process(container.ID)
+		if err != nil {
+			log.Error("error fetching metrics for container %v: %v", container.ID, err)
+			if _, ok := err.(*biz.ErrExitedContainerExpired); ok {
+				log.Debug("skipping exited container older than TTL")
+				continue
+			}
+		}
```

Comment:
```suggestion
		if err != nil {
			if _, ok := err.(*biz.ErrExitedContainerExpired); ok {
				log.Debug(err.Error())
				continue
			}
			log.Error("error fetching metrics for container %v: %v", container.ID, err)
		}
```

This way, `ErrExitedContainerExpired` is logged only once, but with full detail.